openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/RuleForm/RuleForm.tsx

```diff
@@ -100,10 +138,43 @@ const RuleForm: FC<RuleFormProps> = ({ ruleData, setRuleData }) => {
     }
   };
 
+  const fetchPolicyFuntions = async () => {
+    try {
+      const data = await getPolicyFunctions();
+      setPolicyFunctions(data.data || []);
+    } catch (error) {
+      showErrorToast(error as AxiosError);
+    }
+  };
+
+  const handleConditionValidation = async () => {
+    const defaultErrorText = 'Condition is invalid';
+    if (ruleData.condition) {
+      try {
+        const response = await validateRuleCondition(ruleData.condition);
+
+        /**
+         * If request is successful then we will only get response as success without any data
+         * So, here we have to check the response status code
+         */
+        [200, 204].includes(response.status)
```

Comment:
We might have this code from AxiosResponseCode can you chech?

Reply:
Sure.